multiversx_sdk_cli/cli_shared.py

```diff
@@ -180,6 +180,11 @@ def check_options_for_guarded_tx(options: int):
         raise errors.BadUsage("Invalid guarded transaction's options. The second least significant bit must be set.")
 
 
+def check_bytecode_args(args: Any):
+    if not args.bytecode and len(args.bytecode):
```

Comment:
We should declare it as `required` in `_add_bytecode_arg()`. Then, we will not need this validation anymore.

Reply:
Deleted this function.